pandas/core/groupby.py

```diff
@@ -191,6 +191,63 @@
     dtype: int64
     """)
 
+_numeric_operations_doc_template = """
+    Compute %(f)s of group values.
+
+    For multiple groupings, the result index will be a
+    :class:`~pandas.MultiIndex`.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Optional keyword arguments to pass to `%(f)s`.
```

Comment:
Ideally we would list the valid keywords here .. Looking at where min/max/sum etc are create, it's only either `min_count` or either `numeric_only`